dmc_configs/dmc_configs/README.md

```diff
@@ -118,27 +118,27 @@ Consists of 8 experiments:
 - `dist_agent_1lm_randrot_nohyp_x_percent_80p` - 80% threshold
 - `dist_agent_1lm_randrot_nohyp_x_percent_100p` - 100% threshold
 
-### Training (1 experiment):
-- `dist_agent_77obj_1rot_trained`*
-
 This means performance is evaluated with:
 - 77 objects
-- 14 random rotations
-- No sensor noise**
-- No hypothesis-testing**
+- 5 random rotations
+- No sensor noise*
+- No hypothesis-testing*
 - No voting
 
-*Single rotation evaluation due to FLOPs counting overhead, so we will extrapolate total FLOPs to 14 rotations based on this/
-or can compare to a ViT trained on 1 rotation.
+*Due to ViT model comparison.
+
+The main output measure is accuracy and FLOPs as a function of x-percent threshold.
 
-**Due to ViT model comparison.
+### Training (1 experiment):
+- `dist_agent_77obj_1rot_trained`**
 
+**Single rotation evaluation due to FLOPs counting overhead, so we will extrapolate total FLOPs to 14 rotations based on this/or can compare to a ViT trained on 1 rotation.
 
-The main output measure is accuracy and FLOPs as a function of x-percent threshold.
+The main output measure is as a function of whether the ViT or Monty is training.
```

Comment:
I'm not sure I understand it. Wouldn't it be FLOPs?

Reply:
Sorry, typo on my part.